spec/services/moment_keywords_spec.rb

```diff
@@ -0,0 +1,45 @@
+
+describe MomentKeywords do
+    let(:user) { create(:user) }
+    let(:categories) {[FactoryBot.build(:category,
+                                        name: "category name",
+                                        description: "category description."
+    )]}
+    let(:moods) {[FactoryBot.build(:mood,
+        name: "mood-name",
+        description: "mood-description."
+    )]}
+    let(:strategies) {[FactoryBot.build(:strategy,
+        name: "strategy-name",
+        description: "strategy-description."
+    )]}
+    subject(:moment) {FactoryBot.build(:moment,
```

Comment:
As it refers to a class other than the one being tests (`MomentKeywords`), we usually use `let` here

Reply:
Oh, that's what you said over Zoom! Sorry, the audio wasn't good at the time. :)